jquery.timer.js

```diff
@@ -284,31 +284,35 @@
     /**
      * @param {function=} updateFunction
      */
-    function Stopwatch(updateFunction) {
+    function Stopwatch(updateFunction, autostart) {
 
         if (typeof this == "function" || this.init) {
-            return new Stopwatch(updateFunction);
+            return new Stopwatch(updateFunction, autostart);
         }
 
-        this.set(updateFunction);
+        this.set(updateFunction, autostart);
 
         return this;
     }
 
     /**
      * @param {function=} updateFunction
      */
-    Stopwatch.prototype.set = function(updateFunction) {
+    Stopwatch.prototype.set = function(updateFunction, autostart) {
 
         if (typeof updateFunction != "function") {
             return;
         }
 
         this.init = true;
         this.updateFunction = updateFunction;
-        this.startTime = new Date().getTime();
+        this.startTime = 0;
         this.pauseStart = 0;
-        this.setInterval();
+
+        if (autostart != 0) {

```

Comment:
Actually that line is there to keep autostart as the default behavior as currently stopwatch starts automatically? Because if someone does not pass the autostart variable within the constructor, then it won't start by default as the autostart variable will be undefined.